tar_scm.py

```diff
@@ -799,12 +801,12 @@ def cleanup(dirs):
         shutil.rmtree(d)
 
 
-def version_iso_cleanup(version):
+def version_iso_cleanup(version, char=''):
```

Comment:
could you please explain, why an additional parameter is added here as I cannot see where the default (char='') would be changed.

Reply:
It's used as a fake static function variable

Reply:
in the original PR (which okurz kindly rebased into this one) there was at least one point where ```char='.'``` was passed, thus making this change necessary.